corehq/apps/sso/utils/__init__.py

```diff
@@ -0,0 +1,11 @@
+import logging
+
+logger = logging.getLogger('sso')
+
+
+def log_sso_error(message, show_stack_trace=False):
```

Comment:
i think it's better to use

```
logger = logging.getLogger(__name__)
```

at the top of the file and then `logger.info`, `logger.error` as we are doing in the majority of places in the codebase (as well as other places in SSO)

Reply:
Yeah, this was a convention copied from `corehq.apps.accounting.utils` (I initially used that module heavily for references, but that stuff is a bit dated... hopefully this isn't *too much* of a theme in the entire PR). I'll fix this up.